### PR TITLE
feature(landing page):Create an attractive landing-page

### PR DESCRIPTION
**Pull request by Ayesiza**
**What does this PR do?**
- Allows a user to view an attractive landing page

**Description of Task to be completed?**
- Create a landing page with a signup link
- A button that allows a user to navigate available books

**How should this be manually tested?**
- Clone this repository 

**Any background context you want to provide?**
          N/A

**What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/story/show/172040107

**Screenshot**
landing

[Finishes #172040107]

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+     padding: 10px 20px;
+     border-radius: 5px;
+     border: hidden;
+     font-size: larger;
+  }
+  img {
+    width:50%;
+    
+  }
+
```
Comment (jennizalwango) [inline assets/css/style.css]: Good work @Ayesiza and @adutias  kindly add a new line on this page

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            </button>
+            
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">About Us</a>
+                    </li>
+            
+                <!-- <li class="nav-item ">
+                  <a class="nav-link " href="./signup.html">Signup</a>
+              </li> -->
```
Comment (jennizalwango) [inline index.html]: Remove this commented code

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                <a href=""><i class="fa fa-facebook"></i></a>
+                <a href=""><i class="fa fa-github"></i></a>
+            </p>
+        
+    </footer>
+  </div>
+</div>
+</body>
+
+</html>
```
Comment (jennizalwango) [inline index.html]: Add a newline to this file

File: README.md [hunk 4]
```diff
@@ -1,2 +1,38 @@
...
+ Javascript
+
+## ### Author
+
+- Ayesiza Hawah
+- Solomon Adutia
+
+### Acknowledgments
+ 
+ Our thanks goes to Our Team LF Jane Zaluwango, other LFs and Team Strap members
```
Comment (jennizalwango) [inline README.md:38]: Good work about adding a READMe please add a link to  your GH-pages

**Review by jennizalwango — CHANGES_REQUESTED** [review]
Good work
Please work on those comments then ll can approve this PR

File: README.md (hunk 4 above)
Comment (Ayesiza) [reply README.md:38]: I not yet hosted i will add when I host after merging one branch atleast

File: index.html [hunk 5]
```diff
@@ -86,16 +82,13 @@ <h4>Featured Stories</h4>
...
                 <a href=""><i class="fa fa-facebook"></i></a>
                 <a href=""><i class="fa fa-github"></i></a>
             </p>
-        
     </footer>
   </div>
 </div>
 </body>
-
 </html>
```
Comment (jennizalwango) [inline index.html:93]: Please a new line at the end of this file

File: assets/js/app.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  }
+  slideIndex++;
+  if (slideIndex > slides.length) {slideIndex = 1}    
+  for (i = 0; i < dots.length; i++) {
+    dots[i].className = dots[i].className.replace(" active", "");
+  }
+  slides[slideIndex-1].style.display = "block";  
+  dots[slideIndex-1].className += " active";
+  setTimeout(showSlides, 2000); // Change image every 2 seconds
+}
```
Comment (jennizalwango) [inline assets/js/app.js:8]: Add a new line at the end of this file

File: assets/css/style.css [hunk 7]
```diff
@@ -84,8 +81,74 @@ p{
...
+@keyframes fade {
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+
+/* On smaller screens, decrease text size */
+@media only screen and (max-width: 300px) {
+  .text {font-size: 11px}
+}
+/* slides end here */
```
Comment (jennizalwango) [inline assets/css/style.css]: Add a new line at the end of this line

File: assets/js/app.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
```
Comment (jennizalwango) [inline assets/js/app.js]: Please use ES6 syntax for instance instead of using `var` use `Let `or `const` where necessary.

File: assets/js/app.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
```
Comment (jennizalwango) [inline assets/js/app.js]: Please use and mind your naming conventions; For instance, why do you have to use `showSlides` can you please explain why you used it.

**Review by jennizalwango — CHANGES_REQUESTED** [review]
Good work @Ayesiza Please attend to the comments left then you will be good to go.

File: assets/js/app.js (hunk 9 above)
Comment (Ayesiza) [reply assets/js/app.js]: Just a function to make the images slide
For this, I was trying it out and it didn't work for  as I expected forgot to delete it from my file
But I have removed it

**Comment by Ayesiza** [comment]
kindly be informed i have power blackout .I'm cant do any thing since
yesterday

**Pull request merged**